Packages/Env/Sources/Env/DeepLUserAPIHandler.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+import SwiftUI
+
+@MainActor
+public enum DeepLUserAPIHandler {
```

Comment:
I find this a little bit hard to read, I wonder if it can be cleaned when using KeychainSwift.